### DISCUSSION TITLE
Kong plugin form password field

### DISCUSSION THREAD

**Discussion by dashng** [Kong/kong#12392] [Help] [answered]
I need to configure the db password, how to protect the password field value shown as plain text?

**Comment by ms2008** [comment, marked as answer]
Kong allows you to store secret values in external vaults. Then you can reference them in the plugin's configuration fields.

Note that the Kong community version only supports Vault backends implemented by environment variables. For more details. please see https://docs.konghq.com/gateway/latest/kong-enterprise/secrets-management/

**Comment by dashng** [comment]
Thanks @ms2008
